### ISSUE TITLE
Updating Blog section

### ISSUE THREAD

**Issue by YadlaMani** [Nactore-Org/Nacto-Care#41]
I plan to update the blog section with meaningful information instead of random text to make the website more useful and increase overall reliability.
![image](https://github.com/Nactore-Org/Nacto-Care/assets/133841094/08d80e50-82cc-485c-beff-bc608c2744b7)

**Comment by YadlaMani** [comment]
@sunidhisachan @Harshil-Jani pls assign me this issue under gssoc'24

**Comment by Harshil-Jani** [comment]
@YadlaMani Please go ahead with the idea. That would be amazing to see.

**Comment by YadlaMani** [comment]
@Harshil-Jani if possible add level 2 i willl be adding many articles and and willl redirect to the pages when you click on the div

**Comment by YadlaMani** [comment]
@Harshil-Jani pls go trough the changes once the blog section is very good now and update the issue to level 2 if you like the work and worth the effort

**Comment by Harshil-Jani** [comment]
Yes sure. @YadlaMani I would suggest to write one blog but with meaningful insights and I can assign level2. In future you can take up more blogs and all of them at level2. More details in comment of #52

**Comment by YadlaMani** [comment]
@Harshil-Jani add level 2 for the issue to
